Code/GraphMol/ChemTransforms/catch_tests.cpp

```diff
@@ -54,5 +54,25 @@ TEST_CASE("Github #1039", "[]") {
     // conformer to do that using wedging)
     CHECK(MolToSmiles(*pieces) == "*C.[1*]C(O)(F)Cl");
   }
+  SECTION("bond stereo") {
```

Comment:
I think it's worth adding a test where the bond which is broken is to an atom that is *not* one of the StereoAtoms just to ensure that nothing goes wrong there. Here's an example:
```
In [7]: m = Chem.MolFromSmiles('C/C(O)=N/C=C')

In [8]: list(m.GetBondWithIdx(2).GetStereoAtoms())
Out[8]: [2, 4]

In [9]: frag = Chem.FragmentOnBonds(m,[0])
```

Reply:
Will do.

Reply:
just to confirm: that didn't show up in your most recent commit.
are you planning on adding it?

Reply:
@greglandrum should be added now